pyatv/protocols/companion/opack.py

```diff
@@ -97,7 +97,17 @@ def _pack(data, object_list):
 
     # Re-use if in object list, otherwise add it to list
     if packed_bytes in object_list:
-        packed_bytes = bytes([0xA0 + object_list.index(packed_bytes)])
+        object_index = object_list.index(packed_bytes)
```

Comment:
Great that you added these! Some tests are needed, at least for `object_index <= 0xFF` and possibly 0xFF (the other ones might be hard without using a lot of memory).

Reply:
Done.